service/init.go

```diff
@@ -781,20 +844,22 @@ func (pcf *PCF) updateConfig(commChannel chan *protos.NetworkSliceResponse) bool
 				}
 			}
 		}
-		if !minConfig {
+		if minConfig == false {
 			// first slice Created
 			if len(pcfContext.PlmnList) > 0 {
```

Comment:
Add/explain via comment that in what condition this can happen

Reply:
done